src/pilot/templates/DellNfv/numa.pp

```diff
@@ -0,0 +1,47 @@
+# == Class: dellnfv::numa
```

Comment:
Should add the standard licensing header to the top of this file.

Reply:
Added.